### PR TITLE
docs(collaborate): update collaborate doc for local collaboration

### PR DESCRIPTION
**Pull request by KennethBWSong**

### CODE REVIEW

File: docs/vscode-extension/collaborating.md [hunk 1]
```diff
@@ -1,33 +1,39 @@
...
+Multiple developers collaborating on a Teams application should set up their own environments for development including unique [M365 Developer Tenants](https://developer.microsoft.com/en-us/microsoft-365/dev-program) and application registrations in Azure Active Directory. This way each developer runs their code under a different application identity. Our extension provide ability for developers to set up their own local-debug environment. **Every developer can simply press `F5` to start his own application locally.**
+
+If you want to share the local environment, please follow the steps in [Sharing Local Environemnt](#sharing-local-environemnt).
+
+## Sharing Local Environemnt
```
Comment (SLdragon) [inline docs/vscode-extension/collaborating.md:7]: typo: Environemnt -> Environment

File: docs/vscode-extension/collaborating.md [hunk 2]
```diff
@@ -1,33 +1,39 @@
...
+Multiple developers collaborating on a Teams application should set up their own environments for development including unique [M365 Developer Tenants](https://developer.microsoft.com/en-us/microsoft-365/dev-program) and application registrations in Azure Active Directory. This way each developer runs their code under a different application identity. Our extension provide ability for developers to set up their own local-debug environment. **Every developer can simply press `F5` to start his own application locally.**
+
+If you want to share the local environment, please follow the steps in [Sharing Local Environemnt](#sharing-local-environemnt).
```
Comment (SLdragon) [inline docs/vscode-extension/collaborating.md:5]: Same here
typo: Environemnt -> Environment

File: docs/vscode-extension/collaborating.md (hunk 1 above)
Comment (KennethBWSong) [reply docs/vscode-extension/collaborating.md]: Fixed. Thanks!

File: docs/vscode-extension/collaborating.md (hunk 2 above)
Comment (KennethBWSong) [reply docs/vscode-extension/collaborating.md]: Fixed. Thanks!

**Review by SLdragon — APPROVED** [review]

**Review by adashen — APPROVED** [review]

File: docs/vscode-extension/collaborating.md [hunk 3]
```diff
@@ -1,28 +1,34 @@
...
+
+If an identity is to be shared between multiple developers (such as in a staging or integration environment) there are two points of registration which need to be configured to allow multiple developers to run the same app. Azure Active Directory and Teams Developer Portal
 
 As the person creating the project (creator), follow these steps to allow others on your team (collaborators) to collaborate on your application.
 
-## Pre-requisites
+### Pre-requisites
 1. [Creator] Create a project with the Teams Toolkit in the IDE or teamsfx CLI.
 2. [Creator] Start your application locally at least once. This will create an application registration in Azure Active Directory.
 3. [Creator] Go to the [Teams Amdmin Center](https://admin.teams.microsoft.com/policies/app-setup) and select "Global (Org-wide default)". Ensure "Upload custom apps" is turned on.
```
Comment (yanjiang222) [inline docs/vscode-extension/collaborating.md]: typo: amdmin -> admin

**Pull request closed without merging**